src/rf/apps/workers/process.py

```diff
@@ -71,8 +77,13 @@ def start_polling(self):
     def handle_message(self, record):
         # Parse record fields.
         try:
-            job_type = record['eventName']
-            event_source = record['eventSource']
+            if 'eventName' in record:
+                job_type = record['eventName']
+                event_source = record['eventSource']
+            elif 'stage' in record:
+                job_type = record['stage']
+            else:
```

Comment:
Actually, I don't think we need to change this now. Future enhancement? :grinning: